routes/userRoutes.js

```diff
@@ -6,7 +6,7 @@ const userRouter = express.Router();
 
 userRouter
   .route('/')
-  .get(userController.getUser)
```

Comment:
hi @meeklife the convention is to use the IDs to fetch individual resources. So, in the setup, '/:id' is meant to handle requests to retrieve a single user based on their unique ID.  We're not using the unprotected '/' route to view a single user.